client/src/www/ui_components/app-root.js

```diff
@@ -712,6 +725,25 @@ export class AppRoot extends mixinBehaviors(
     this.$.addServerView.open = true;
   }
 
+  showErrorDetails(errorDetails) {
+    // <if-messages> hides the error details from view but does not remove it
```

Comment:
I feel it's acceptable to have English error detail dialog if translations are not available, because the error detail itself is not translated; then we don't need to change anything related to `if_messages`.

But I'll let @sbruens to make the decision.